src/bugs/socket_buffer.md

```diff
@@ -0,0 +1,38 @@
+# Too small socket buffer a.k.a. packet loss in a local network
+
+## The problem
+
+At some point in implementing simulcast, we started noticing very strange statistics in the webrtc-internals.
+On some of our machines, web browsers connected via our WebRTC SFU were dropping a lot of RTP packets and video frames.
+The QoE was terrible.
+The client was displaying something around 3 FPS.
+It didn't matter whether we were testing in a local or remote network.
+At the beginning, we thought that this is some kind of packet loss but we couldn't get how it might be possible
+in the local network.
+We entirely turned off simulcast, decreased resolution, changed codecs and downgraded web browsers.
+Nothing helped.
+The only thing we noticed was that the problem was not occurring on pretty strong machines like those with the 12th gen i9 processors or when we disabled TWCC.
```

Comment:
Actually, I think the issue here was mostly related to webcam quality :stuck_out_tongue: